docs/en/Best-Practices/PostgreSQL-Integration.md

```diff
@@ -0,0 +1,61 @@
+﻿## Entity Framework Core PostgreSQL Integration
+
+> See [Entity Framework Core Integration document](../Entity-Framework-Core.md) for the basics of the EF Core integration.
+
+### EntityFrameworkCore Project Update
+
+- In `Acme.BookStore.EntityFrameworkCore` project replace package `Volo.Abp.EntityFrameworkCore.SqlServer` with `Volo.Abp.EntityFrameworkCore.PostgreSql` 
+- Update to use PostgreSQL in `BookStoreEntityFrameworkCoreModule`. Example:
+
+````C#
+[DependsOn(
+        //code omitted for brevity
+		
+		/* This was updated from AbpEntityFrameworkCoreSqlServerModule */
```

Comment:
This seems to be more concise, such as: Replace the `AbpEntityFrameworkCoreSqlServerModule `with the `AbpEntityFrameworkCorePostgreSqlModule`.

Similarly: `options.UsePostgreSql()`  : )